doc/source/api/rnaquantification.rst

```diff
@@ -0,0 +1,83 @@
+
+Case 6: Obtain quantification data for non-read-based RNA experiments (MicroArrays)
+
+User desires:
+Discover and retrieve feature level quantification data that is derived from non-read-based sources such as the large sources of microarray-based expression data.  The quantification API needs to be source agnostic and allow for a general linking of quantity to feature.  It must be flexible and not lock the results to a reads/sequencer data collection model.  There should be no required data source or metadata fields that are specific for a given data collection method.
+
+
+---------------------------------------
+Annotation Design - RNA Considerations
+---------------------------------------
+
+Read data derived from RNA samples can differ from genomic read data due to the presence of non-genomic sequences.  An example would be a read that spans a splice junction.  It describes a contiguous sequence of reads, but a dis-continuous genomic region due to the missing intron.  Feature level read assignment is further complicated by the existence of multiple splice isoforms.  A read that can be definitely assigned to a particular feature (an exon in this case) may still not be definitely assigned to a particular transcript if multiple transcript share that exon.  The annotation API needs to be able to report assignment at the feature level as well as aggregate assignment at the transcript or even the whole gene level if assignment is not more specific than that.
+
+Splicing (other post-transcriptional modifications?) can occur with degrees of complexity.  A ‘typical’ splice will result in a mature transcript with exon in positional (numerical) order in a head-to-tail orientation.  Back splicing (tail-to-head) can result in transcripts with the exon order reversed (1-3-2-4 instead of 1-2-3-4) and even circular RNA.  The exon order in a transcript as well as the orientation of the splice should be discoverable via the API.  In a more general case, the API should allow child features to have an ordered relationship.
+
+The annotation API needs to also be flexible enough to handle multiple references in the same gene or transcript.  This is needed to cover the cases of fusion genes or inter-chromosomal translocations.
+
+--------------------------
+RNA Quantification Schema
+--------------------------
+
+The RNA Quantification Schema is designed around a quantification analysis.  Each set of feature quantifications describes the results of running an analysis ona set of input data.
+
+The RnaQuantification identifies the experiment and describes the analysis pipeline used as well as the input dataset and annotations, if any, used.  ReadCounts and Characterization are optional API elements that further describe RNASeq analyses.
```

Comment:
I don't see mention of characterization elsewhere, though read counts are mentioned.